docs/00-preamble/copyright.adoc

```diff
@@ -18,15 +17,22 @@ Lizenzverträge, die alles umfassend regeln, sind vorhanden.
 3.	Falls Sie weder unter die Kategorie 1. noch unter die Kategorie 2. fallen, aber dennoch die Dokumente und/oder Lehrpläne nutzen möchten, nehmen Sie bitte ebenfalls Kontakt unter info@isaqb.org zum iSAQB e. V. auf.
 Sie werden dort über die Möglichkeit des Erwerbs entsprechender Lizenzen im Rahmen der vorhandenen Lizenzverträge informiert und können die gewünschten Nutzungsgenehmigungen erhalten.
 
+.Wichtiger Hinweis
+****
 *Grundsätzlich weisen wir darauf hin, dass dieser Lehrplan urheberrechtlich geschützt ist.
 Alle Rechte an diesen Copyrights stehen ausschließlich dem International Software Architecture Qualification Board e. V. (iSAQB^(R)^ e. V.) zu.*
+****
+// end::DE[]
```

Comment:
This is end-tag appears further down below, and will thus break/cause something to disappear. This tag has to be removed (as you did in the foundation repository)

Reply:
right you are. Sorry, my mistake.

Reply:
I added another commit - please re-consider merging - thx